config/initializers/rollout.rb

```diff
@@ -13,3 +13,11 @@
 Rollout::UI.configure do
   instance { ROLLOUT }
 end
+
+def ispf_in_stealth_mode_for_group?(user_group)
```

Comment:
Would it be a pain to call these `show(/hide)_ispf_for_group?` and `show(/hide)_ispf_for_user?` There's something about my brain that requires a few re-reads `in_stealth_mode` to determine which way the boolean is pointing, but I can get over it if it's a faff.

Reply:
Not a faff, and it's a fair comment!